webrob/pages/QA.py

```diff
@@ -15,6 +15,7 @@
 
 @app.route('/QA')
 def render_QA_page():
+    neem_manager = NEEM_Manager()
```

Comment:
this looks like an error. NEEM Manager is supposed to be a singleton with only one global instance.